.github/workflows/update_rubocop.yml

```
@@ -0,0 +1,38 @@
+
+name: Update Rubocop
+
+on:
+  # Can be triggered manually from the GH interface or via a dispatches POST
+  # https://docs.github.com/en/actions/managing-workflow-runs/manually-running-a-workflow
+  workflow_dispatch:
+  schedule:
+  # * is a special character in YAML so you have to quote this string
+  # Every sunday at 10 am
+  - cron: "0 10 * * 0"
+
+jobs:
+  update_rubocop:
+    if: (github.repository_owner == 'sanger') # On the sanger fork only
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Ruby
+      uses: ruby/setup-ruby@v1
+      env:
+        BUNDLE_WITHOUT: 'test development default'
```

```suggestion
        BUNDLE_WITHOUT: 'test:development:default'
```
Seems they might need to be separated with colons [here](A :-separated list of groups whose gems bundler should not install.)

---

What does it bundle with, if not any of them? Just the `lint` group?

---

Yep, just the lint group. Keeps the install time right down low.
Where did you get that colon separation doc from? Space separation is definitely working (Sequencecape builds would fail completely if it wasn't) but it might be that they are in the process of switching from one to another

---

https://bundler.io/v1.16/bundle_config.html search `without (BUNDLE_WITHOUT): A :-`

---

Thanks... I wonder why its working? Will definitely make that change.